frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { match as RouterMatch } from 'react-router';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { Alert, Button } from '@patternfly/react-core';
+import { StorageClassResourceKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { history } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { fetchK8s } from '@console/internal/graphql/client';
+import { ClusterServiceVersionModel, SubscriptionModel } from '@console/operator-lifecycle-manager';
+import { NO_PROVISIONER, LSO_NAMESPACE } from '../../../constants';
+import { CreateOCS } from './install-lso-sc';
+import { scResource } from '../../../constants/resources';
+
+import './attached-devices.scss';
+
+export const CreateOCSOnAttachedDevices: React.FC<CreateOCSOnAttachedDevicesProps> = ({
+  match,
+}) => {
+  const { ns, appName } = match.params;
+  const [isNoProvSCPresent, SetIsNoProvSCPresent] = React.useState<boolean>(false);
```

Comment:
> 

Well, the variable name is this because its for checking whether no provisioner SC is present or not. `isNoProvisionerSC` name looks more like whether a SC is a no provisioner or not.